### PR TITLE
Mario Party Cheats Update

### PR DESCRIPTION
**Pull request by NoraTheGamer**

### CODE REVIEW

**Comment by NoraHanegan** [comment]
Update of my older PR #8705 that I polished up and added a few more codes.

**Comment by JosJuice** [comment]
It looks like you've changed the line endings to CRLF. Please keep them as LF.

**Comment by NoraHanegan** [comment]
Thank you for pointing that out

**Comment by JMC47** [comment]
You should be squashing commits so this is only a single commit in the end.

**Comment by NoraHanegan** [comment]
ill do it but cant GitHub squash commits with a button now

**Comment by NoraHanegan** [comment]
Any Updates @JosJuice

**Comment by NoraHanegan** [comment]
This PR adds some new gecko codes to improve the quality of Mario Party on Dolphin. This PR also removes some old codes that are broken and replaced with better similar functioning ones. The codes placed here originally were done by me around a year ago and i would like to revise them. The netplay specific codes are being removed and breaking them down to 3 distinct categories.

**Comment by Rumi-Larry** [comment]
I would name the cheats that remove the intro screens to be just that "Remove Minigame Intro Screens" You can change the word remove for "skip". Booting can take a variety of meanings, like booting the game itself or a netplay session.

**Comment by NoraHanegan** [comment]
> Screens

Because it is not minigame screens. This code specifically skips the hudson screen and stuff like that.

**Comment by Rumi-Larry** [comment]
> 
> 
> > Remove Minigame Intro Screens
> 
> Because it is not minigame screens. This code specifically skips the hudson screen and stuff like that.

Yeah, I'm dumb...

File: Data/Sys/GameSettings/GMPE01.ini [hunk 1]
```diff
@@ -7,114 +7,137 @@
...
 0A1E67F0 00000030
 0018FCFC 00000014
 
-$Press Z+L To Be On Turn 1
+$Press Z+L To Be On Turn 1 [Datel]
 0A1E67F0 00000050
 0018FCFC 00000001
 
 [Gecko]
-$Netplay Community Settings
```
Comment (leoetlino) [inline Data/Sys/GameSettings/GMPE01.ini:107]: Why was this removed?

File: Data/Sys/GameSettings/GMPE01.ini [hunk 2]
```diff
@@ -7,114 +7,137 @@
...
+$QOL - Allow Digital Presses for Map Screen [gamemasterplc]
 c2086774 00000002
 70000020 2c000020
 60000000 00000000
 c2073c7c 00000002
 735a0020 281a0020
 60000000 00000000
+*This cheat fixes a bug where only analog inputs work while using the triggers.
+
+*QOL - Faster Boot Time [Ralf]
```
Comment (leoetlino) [inline Data/Sys/GameSettings/GMPE01.ini:118]:
```suggestion
$QOL - Faster Boot Time [Ralf]
```
?

File: Data/Sys/GameSettings/GP7E01.ini [hunk 3]
```diff
@@ -116,38 +655,351 @@ $Board - Neon Heights: Balance Changes
...
 
-$Board - Pyramid Park: Free Chomp Rides
-204E10B4 7C03D000
-044E10B0 386003E7
-044E1114 386003E7
-044E14EC 386003E7
-044E17A4 38A00000
-E2000001 80008000
+$Board - Pyramid Park: Chomps Sell Stars, Players Start with 0 Stars
+204ddf34  4180ffb8
```
Comment (leoetlino) [inline Data/Sys/GameSettings/GP7E01.ini:678]: There are two spaces between each column in this code. Is this intended?

**Comment by leoetlino** [comment]
Could you fix your PR description and commit message to actually describe the changes?

> This PR adds some new gecko codes to improve the quality of Mario Party on Dolphin. This PR also removes some old codes that are broken and replaced with better similar functioning ones. The codes placed here originally were done by me around a year ago and i would like to revise them. The netplay specific codes are being removed and breaking them down to 3 distinct categories.

This description you provided sounds good but it should be part of your commit message.

Also, in the future, consider splitting your changes into several independent commits. Having one large monolithic commit with thousands of modified lines with little explanation of what the changes do makes people less willing to review your PR.

**Comment by NoraTT** [comment]
I did originally i was told to squash it

File: Data/Sys/GameSettings/GP7E01.ini (hunk 3 above)
Comment (NoraTT) [reply Data/Sys/GameSettings/GP7E01.ini:678]: nope

**Comment by leoetlino** [comment]
I believe you're misunderstanding. Having a big monolithic commit + fixup commits like "fix line endings" is utterly useless and the fixup commits should definitely be squashed into the relevant commits.

What I meant by splitting your changes is having e.g. one commit that adds the author information, one that adds descriptions to existing codes, another that adds the new codes, etc.

File: Data/Sys/GameSettings/GMPE01.ini (hunk 1 above)
Comment (NoraTT) [reply Data/Sys/GameSettings/GMPE01.ini:107]: because it just added the previous commits and no one in the netplay community toggles it

**Comment by NoraTT** [comment]
Oh i see, want me to redo this PR for yall

**Comment by leoetlino** [comment]
That'd be great as it'd make the PR more pleasant to review.

**Comment by NoraTT** [comment]
If you are curious the differences between the sections are as such. QOL which is at the top is stuff that is just useful like speeding up animations and stuff. 

Board is for tweaking stuff that are part of the board like changing how Pyramid Park operates. 

Extra and Game are a little tricky to explain but to easily explain Game is stuff to **tweak** the board and Extras is ASM hacks to completely change the gameplay

Minigame is to tweak minigames and Minigame Replacement is to replace minigames from ever appearing.

**Comment by NoraTT** [comment]
Is this what you are looking for?

**Comment by leoetlino** [comment]
Yep, that's a lot nicer, thanks! Just FYI, there's a typo in one of your commit messages ("Remove broken cheats and merge some togeather" -> together). Also, I'd prefix your commit titles with `GameINI: ` (e.g. GameINI: Mario Party 4 - Add Authors to Cheats) to clearly indicate you're editing the game INIs.

**Comment by NoraTT** [comment]
ahh okay

**Comment by NoraTT** [comment]
@leoetlino Requesting another review. I was the original author of these codes for clarification

File: Data/Sys/GameSettings/RM8E01.ini [hunk 4]
```diff
@@ -1,16 +1,19 @@
-# RM8E01 - Mario Party 8
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:1]: You shouldn't be removing this opening line.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 5]
```diff
@@ -116,38 +655,180 @@ $Board - Neon Heights: Balance Changes
...
+38800000 908d8ac0
+908d8ac4 38600001
+3c808000 6084e8f4
+7c8903a6 4e800421
+60000000 00000000
+c204a98c 00000002
+7c9f2378 7ca50e70
+60000000 00000000
+*Makes the game run at 30Hz but still feel like 60FPS. Great for people with bad PCs
+*Can break certain mingames
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP7E01.ini:772]:
```suggestion
*Can break certain minigames
```

File: Data/Sys/GameSettings/GMPE01.ini [hunk 6]
```diff
@@ -229,171 +263,114 @@ e2000001 80008000
...
-0406e314 c02281f0
-*This cheat speeds up game animations.
-
-$Game Mechanics - Free Lottery Game
+$Game Mechanics - Free Lottery Game [Celerizer]
 0407b28c 38600005
 0407bd04 38800000
-*This cheat makes the lottery free.
+*Makes the lottery cost nothing instead of the usual 5 coins.
+*You to have 5 coins but you do not lose them.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GMPE01.ini:272]:
```suggestion
*You need to have 5 coins but you do not lose them.
```

File: Data/Sys/GameSettings/GP5E01.ini [hunk 7]
```diff
@@ -464,109 +464,130 @@ $Press L+X: Player Stops Moving
...
 0446A0B0 2C030000
 0446A138 2C040000
 0446A294 2C030000
 0446A308 38800000
 E2000001 80008000
-*This cheat makes the rainbow bridges cost nothing instead of the usual 5 coins.
+*Rainbow Bridges from Rainbow Dream cost nothing instead of the usual 5 coins.
 
-$Board - Sweet Dream: Disable Topmost Happenning Space
+$Board: Sweet Dream - Disable Topmost Happenning Space [gamemasterplc]
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP5E01.ini:571]:
```suggestion
$Board: Sweet Dream - Disable Topmost Happening Space [gamemasterplc]
```

File: Data/Sys/GameSettings/GP5E01.ini [hunk 8]
```diff
@@ -464,109 +464,130 @@ $Press L+X: Player Stops Moving
...
 2045B2F4 F3E10148
 0445B2E4 4E800020
 E2000001 80008000
-*This cheat disables the topmost Happenning space in Sweet Dream.
+*Disables the topmost Happenning space in Sweet Dream.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP5E01.ini:575]:
```suggestion
*Disables the topmost Happening space in Sweet Dream.
```

File: Data/Sys/GameSettings/GP5E01.ini [hunk 9]
```diff
@@ -579,365 +600,199 @@ f6000001 80008180
...
+*Last 5 Turns Event is always 5 Star Spaces
+
+$Mechanics - Last 5 Turns is Capsule Spaces on Every Space [gamemasterplc]
+20288860 00000076
+045152BC 00000001
+E2100000 80008000
+20288860 00000078
+044FE3BC 00000001
+E2000001 80008000
+*Last 5 Turns Event is always All Spaces Have Capsule Events
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP5E01.ini:661]:
```suggestion
*Last 5 Turns Event Forces All Spaces to Have Capsule Events
```

File: Data/Sys/GameSettings/GP5E01.ini [hunk 10]
```diff
@@ -579,365 +600,199 @@ f6000001 80008180
...
+*Last 5 Turns Event Never Happens
+
+$Mechanics - Last 5 Turns is Red Spaces are Bowser Spaces [gamemasterplc]
+20288860 00000076
+045152BC 00000002
+E2100000 80008000
+20288860 00000078
+044FE3BC 00000002
+E2000001 80008000
+*Last 5 Turns Event is always Red Spaces turn into Bowser Spaces
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP5E01.ini:678]:
```suggestion
*Last 5 Turns Event Forces Red Spaces to Turn into Bowser Spaces
```

File: Data/Sys/GameSettings/GP5E01.ini [hunk 11]
```diff
@@ -579,365 +600,199 @@ f6000001 80008180
...
+14000004 38600000
+E0000000 80008000
+*This cheat makes it so Capsules can be used on Yourself for Free instead of the usual 5 coins.
+
+$Minigame: Beam Team - 20 Second Timer [Airsola]
+20288860 00000025
+28467572 00000708
+02467572 000004b0
+e2000002 80008000
+*This cheat makes it so Pop Star Piranhas is more balanced.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP5E01.ini:699]:
```suggestion
*This cheat Balances the Pop Star Piranhas Minigame.
```

File: Data/Sys/GameSettings/GP5E01.ini [hunk 12]
```diff
@@ -579,365 +600,199 @@ f6000001 80008180
...
-EC22082A 00000000
-C2472924 00000005
-EC24002A 3C803FAB
-9081FFFC C041FFFC
-EC220072 3C80C2C0
-9081FFFC C041FFFC
-EC22082A 00000000
-E2000001 80008000
-*This cheat makes the game appear in widescreen.
+e2000001 80008000
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP5E01.ini:798]: newline

File: Data/Sys/GameSettings/GMPE01.ini [hunk 13]
```diff
@@ -229,171 +263,114 @@ e2000001 80008000
...
-EC420072 3C8042C0
-90810044 C0210044
-EC220828 00000000
-C243C964 00000003
-C03B0010 3C803FAB
-90810044 C0410044
-EC220072 00000000
-E2000001 80008000
-*This cheat makes the game appear in widescreen.
+*Makes this minigame easier to play by just requiring the use of L.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GMPE01.ini:376]: newline

File: Data/Sys/GameSettings/GP6E01.ini [hunk 14]
```diff
@@ -95,157 +686,238 @@ C2183590 00000002
...
-EC00E824 00000000
-0414E760 C3A30000
-041932E0 388002D0
-042C14B8 3FCCCCCD
-04151324 38600001
-*This cheat makes the game appear in widescreen.
+$Minigame Replacement - Wrasslin' Rapids ➜ Insectiride [Nora]
+28265ba8 00000013
+02265ba8 00000003
+e2000001 80008000
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP6E01.ini:923]: newline

File: Data/Sys/GameSettings/GP7E01.ini [hunk 15]
```diff
@@ -157,280 +838,241 @@ c24e6288 00000006
...
+$Mechanics - Mic Minigames Have A 10% Chance Of Appearing Instead Of 20% [gamemasterplc]
+042f7a04 41200000
+*Less Mic Minigames and see more standard minigames.
 
-$Game Mechanics - Slow 'Shroom Orb Rolls 1-5
+$Mechanics - Microphone Always Off [gamemasterplc]
+04098fc4 3800002c
+04098fc8 980317e6
+04098fcc 38600000
+*Microphone game setting stay off while playing Mario Party 7.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP7E01.ini:951]:
```suggestion
*Microphone game setting stays off while playing Mario Party 7.
```

File: Data/Sys/GameSettings/GP7E01.ini [hunk 16]
```diff
@@ -157,280 +838,241 @@ c24e6288 00000006
...
+
+$Mechanics - Orbs Can Only Be Placed On Your Current Space [gamemasterplc]
+041e7128 60000000
+*Orb is severly balanced.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP7E01.ini:955]:
```suggestion
*Orb is more aggressively balanced.
```
I think this is what you mean here.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 17]
```diff
@@ -157,280 +838,241 @@ c24e6288 00000006
...
+
+$Minigame Replacement - Warp Pipe Dreams ➜ Apes of Wrath
+28291558 00000031
+02291558 00000038
+e2000001 80008000
+
+$Minigame Replacement: - Pogo-a-Go-Go ➜ Balloonatic
+28291558  00000020
+02291558  00000016
+e2000001  80008000
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP7E01.ini:1078]: newline

File: Data/Sys/GameSettings/RM8E01.ini [hunk 18]
```diff
@@ -137,220 +140,240 @@ de000000 80008180
...
+de000000 80008180
+30109890 80bf01a8
+d210987c 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
+e0000000 80008000
+*Makes the Cashzap Candy steal coins rather then destroying.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:292]:
```suggestion
*Makes the Cashzap Candy steal coins rather then destroying them.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 19]
```diff
@@ -137,220 +140,240 @@ de000000 80008180
...
+30015870 4e800020
+14015874 4e800020
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+30010778 4e800020
+1401077c 4e800020
+e0000000 80008000
+*Disables the overpowered Lucky Spaces.
+*Disables certain happening spaces dueo to the level of assembly needed for this patch.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:346]:
```suggestion
*Disables certain happening spaces due to the level of assembly needed for this patch.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 20]
```diff
@@ -368,301 +391,135 @@ de000000 80008180
...
+de000000 80008180
+30160f24 408202e4
+d216103c 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def0214 b1ee0026
+901f0180 00000000
+e0000000 80008000
+*Makes the Thwomp Candy Steals coins rather then Destroying
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:496]:
```suggestion
*Makes the Thwomp Candy Steals coins rather then Destroying them.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 21]
```diff
@@ -671,20 +528,52 @@ d200abd8 00000003
...
-48000000 800030C8
-DE000000 80008180
-3000B34C 881E0000
-1400B348 3BA00064
-E0000000 80008000
-*This cheat makes it so Surfs Way Up is more balanced
+$Minigame Replacement - You're the Bob-omb ➜ Lava Lobbers [gamemasterplc]
+282287CC 00000037
+022287CC 0000002E
+E2000001 80008000
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:579]: newline

**Review by PatrickFerry — CHANGES_REQUESTED** [review]

File: Data/Sys/GameSettings/RM8E01.ini (hunk 21 above)
Comment (NoraTT) [reply Data/Sys/GameSettings/RM8E01.ini]: Git was forcing to change to CRLF had to reverse it sorry for the trouble

File: Data/Sys/GameSettings/GP7E01.ini [hunk 22]
```diff
@@ -5,109 +5,653 @@
...
-C2047400 00000002
-807F0050 906D0000
+*Makes it so you can taunt during your turn.
+
+$QOL - Instant Text Display [gamemasterplc]
+04050a70 38800000
+*Makes it so text is displayed instantly.
+
+$QO
+ - Show Controller Port Number of Who Paused in Mini-Games [gamemasterplc]
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/GP7E01.ini:620]: This should be on one line

**Review by PatrickFerry — COMMENTED** [review]
You can then use git reset soft to undo the master branch merge and the second commit with the same title while preserving your changes and then commit again.

checkout the correct branch -> master in this case.
Then you can run something like this and re-commit
`git reset --soft 9cc46470a983f718bbe389ac66a052de5ef35e00`

**Comment by JMC47** [comment]
There's a few typos here and there, however the actual codes seem fine.  I tested a few random ones just to see if anything broke.

If the typos are fixed and the commit history is fixed, I'm fine with the actual content of this change.

"*Access all board events regarless of Mega / Mini Mushrooms." as an example of a typo.

**Comment by NoraTT** [comment]
> If the typos are fixed and the commit history is fixed, I'm fine with the actual content of this change.

You want me the squash the commits

**Comment by NoraTT** [comment]
> You can then use git reset soft

Mind if i just push a new commit and squash them all after

**Comment by PatrickFerry** [comment]
> > You can then use git reset soft
> 
> Mind if i just push a new commit and squash them all after

I don't think we want them all squashed. Ask on IRC, but I would recommend only combining the last 3 together as it stands.

File: Data/Sys/GameSettings/RM8E01.ini [hunk 23]
```diff
@@ -6,11 +6,27 @@
...
 [Gecko]
-$Netplay Community Settings
-282287CC 00000032
-022287CC 0000003A
-E2000001 80008000
+# Add gecko cheats here.
+
+$!!! - README
+*Mario Party 8 Cheats can cause the game to stop saving. Be Warned
+*Mario Party 8 Cheats can also cause unintended consiquences due to to hacky nature of specific codes.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:17]:
```suggestion
*Mario Party 8 Cheats can also cause unintended consequences due to to hacky nature of specific codes.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 24]
```diff
@@ -137,220 +153,243 @@ de000000 80008180
...
+1401a2d8 60000000
+1401a160 2c040000
+e0000000 80008000
+*Cannon usage is free in the board DK's Treetop Temple
+
+$Board: Goomba's Booty Boardwalk - Remove Dolphin Shuttles [gamemasterplc]
+2057bc40 437f0000
+0457bc3c 42700000
+e2000001 80008000
+*A long teadious path forms in the board Goomba's Booty Boardwalk.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:227]:
```suggestion
*A long tedious path forms in the board Goomba's Booty Boardwalk.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 25]
```diff
@@ -137,220 +153,243 @@ de000000 80008180
...
+de000000 80008180
+3003f0e0 387f01c0
+1403f0dc 60000000
+1403f04c 2c03000a
+e0000000 80008000
+*Whomp toll is free in the board King Boo's Haunted Hideaway
+
+$Board - King Boo's Haunted Hideaway: Map Always Revealed [gamemasterplc]
+042287bc ffffffff
+*King Boo's Haunted Hideaway becomes not that secret.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:240]:
```suggestion
*King Boo's Haunted Hideaway becomes less secret.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 26]
```diff
@@ -137,220 +153,243 @@ de000000 80008180
...
+30109890 80bf01a8
+d210987c 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
-*This cheat makes it so the player on port 1 will be turned into a computer. This can be useful for games when the player with port 1 drops.
+*Cashzap Candy steals coins rather then destroying.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:308]:
```suggestion
*Cashzap Candy steals coins rather then destroying them.
```

File: Data/Sys/GameSettings/RM8E01.ini [hunk 27]
```diff
@@ -368,301 +407,135 @@ de000000 80008180
...
+de000000 80008180
+30160f24 408202e4
+d216103c 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def0214 b1ee0026
+901f0180 00000000
+e0000000 80008000
+*Thwomp Candy steals coins rather then destroying.
```
Comment (PatrickFerry) [inline Data/Sys/GameSettings/RM8E01.ini:512]:
```suggestion
*Thwomp Candy steals coins rather then destroying them.
```

**Review by PatrickFerry — CHANGES_REQUESTED** [review]
If you could make these typo fixes.

Then interactively rebase so that these new changes that correct typos etc instead replace the same lines in the older commit that introduced the lines. I hope that makes sense.

File: Data/Sys/GameSettings/RM8E01.ini (hunk 26 above)
Comment (leoetlino) [reply Data/Sys/GameSettings/RM8E01.ini:308]: rather th**a**n

**Comment by NoraTT** [comment]
> If you could make these typo fixes.
> 
> Then interactively rebase so that these new changes that correct typos etc instead replace the same lines in the older commit that introduced the lines. I hope that makes sense.

These were just in the last commit since I was trying to redo all the descriptions to match. I still think all the commits should be squashed but its up to yall.

**Comment by NoraTheGamer** [comment]
Updates?

**Comment by leoetlino** [comment]
That last giant commit basically defeats the whole point of splitting your changes into small, independent, easy-to-review commits :/

The typo fixes should go into the relevant commits, but that's not the most problematic aspect of the last commit. Why are brand new codes being added there?

**Comment by NoraTheGamer** [comment]
Can I please just squash all the commits. I created the original pull request and never had to seperate changes there. We really need these codes merged

**Comment by Rumi-Larry** [comment]
At this point it would be better to do a third try of this pr, this time with the commits organized as they are asking you to.

**Comment by Rumi-Larry** [comment]
It's unnecessary to separate the individual changes for each individual ini. You either need it to integrate all the changes for the individual ini's, or to use different commits for the different changes across the files. The first way seems better suited. I'm unsure if this pr can be salvaged to be reformatted like that, you may need to create a third iteration and definitely don't create a commit just integrating all other commits, that's taken care of by the build process.

**Comment by JosJuice** [comment]
> I'm unsure if this pr can be salvaged to be reformatted like that, you may need to create a third iteration

It is always possible to salvage a PR without having to create a new one. (Well, except for the case where you selected the wrong branch as the merge target, I think... But that's not relevant here.)

**Comment by NoraTheGamer** [comment]
Okay should hopefully be mergable

**Comment by NoraTheGamer** [comment]
Any updates @JosJuice @Rumi-Larry @leoetlino

**Comment by Rumi-Larry** [comment]
@NoraTheGamer I don't have merging privileges, but I would merge it as is.

**Comment by JMC47** [comment]
@dolphin-emu-bot rebuild

File: Data/Sys/GameSettings/RM8E01.ini (hunk 26 above)
Comment (JMC47) [reply Data/Sys/GameSettings/RM8E01.ini:308]: It's a super minor grammar error so I think we can let it slide for now instead of possibly messing up the commit history.

**Review by JMC47 — APPROVED** [review]
This greatly organizes and fixes up cheats along with crediting the authors of cheats.  It overall improves the grammar though there's a few very minor things.  I didn't test every single cheat code but most of these codes already existed.
